demos/toolbar/default-toolbar.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
SGTM. I will make changes to the demo showing the following:
- Default
- Fixed to top
- Add checkbox, so it can change RTL on both demos as well

Sound good?